packages/wdio-cli/src/constants.ts

```diff
@@ -423,6 +424,19 @@ export const QUESTIONNAIRE = [{
     message: 'Do you want to add a plugin to your test setup?',
     choices: SUPPORTED_PACKAGES.plugin,
     default: []
+}, {
+    type: 'checkbox',
+    name: 'services',
+    message: 'Do you want to add a service to your test setup?',
+    choices: SUPPORTED_PACKAGES.service.slice(
+        SUPPORTED_PACKAGES.service.findIndex(({ name }) => name ==='browserstack')).concat(
+        SUPPORTED_PACKAGES.service.slice(0, SUPPORTED_PACKAGES.service.findIndex(({ name }) => name ==='browserstack'))),
+    // @ts-ignore
+    default: [SUPPORTED_PACKAGES.service.find(
+        /* istanbul ignore next */
+        ({ name }) => name === 'browserstack').value],
+    when: /* istanbul ignore next */ (answers: Questionnair) => answers.backend === 'In the cloud using BrowserStack',
+    validate: /* istanbul ignore next */ (answers: string[]) => validateServiceAnswers(answers)
```

Comment:
There is no need to add a second entry for selecting services. Just merge it with the one below and update the selected defaults properly.

Reply:
Done